source/plugin/data/serialization.rst

```diff
@@ -123,3 +125,56 @@ from persistent files using the :doc:`Configurate Library <../configuration/inde
         final DataView container = translator.translate(node);
         return deserializeHealth(container);
     }
+
+
+DataFormat
+==========
+An alternative to DataTranslators is to use :javadoc:DataFormat, which allows you to store a ``DataContainer`` as a HOCON, JSON or NBT file. You can also retrieve data using ``DataFormat``.
+
+This is very useful if you're for example using a database to store information, as you can then serialize any ``DataContainer`` to json format.
```

Comment:
There are a few instances in your commit still where you need to update "json" to JSON. Also, this sentence doesn't seem like it fits well - it might be better to say something like:

> For example, you can use ``DataFormat`` to create a JSON representation of a ``DataContainer``, which can then be easily stored in a database as JSON. ``DataFormat`` can then be used to rebuild the ``DataContainer`` from this JSON when required.

It's possible it could be worded better than that, but this sentence doesn't read well as it is.